plugins/modules/win_dns_zone.py

```diff
@@ -80,6 +80,13 @@
       - At least one server is required.
     elements: str
     type: list
+  computer_name:
```

Comment:
I'm torn as to whether this should be `server` or `computer_name`. I know the underlying cmdlets use `-ComputerName` but we've already set a precedent for `server` with the AD modules to explicitly select the AD host to target when managing the object.

I personally lean towards making this `server` to make it uniform with what we already do but interested to hear what you think.

Reply:
Yeah I'd agree, server is easier to understand - When I first read the docs for the module, i didn't get it and it wasn't until i got the powershell cmd working I realized what to do, so yep can make those changes, as well as to win_dns_record